pages/common/astyle.md

```diff
@@ -0,0 +1,24 @@
+# astyle
+
+> Source code indenter, formatter, and beautifier for the C, C++, C# and Java programming languages.
+> A copy of the original file is created with an dot orig appended to the original file name.
+
+- Apply the default style of 4 spaces per indent and no formatting changes:
+
+`astyle {source_file}`
+
+- Apply the java style with attached braces:
+
+`astyle  --style=java {path/to/file}`
+
+- Apply the allman style with broken braces:
+
+`astyle --style=allman {path/to/file}`
+
+- Apply a custom indent using spaces. Choose between 2 and 20 spaces:
```

Comment:
What happens if I give > 20, will the command not work ?

Reply:
If you pass more than 20 astyle will error out. 
```
vagrant@vagrant-ubuntu-trusty-64:~$ astyle --indent=spaces=21 SuperBool.java
Invalid command line options:
indent=spaces=21

For help on options type 'astyle -h'

Artistic Style has terminated
```